apps/astarte_data_updater_plant/lib/astarte_data_updater_plant/health/queries.ex

```diff
@@ -18,11 +18,12 @@
 
 defmodule Astarte.DataUpdaterPlant.Health.Queries do
   require Logger
+  alias Astarte.DataUpdaterPlant.Config
 
   def get_astarte_health(consistency) do
     query = """
     SELECT COUNT(*)
-    FROM astarte.realms
+    FROM  #{Config.astarte_instance_id!()}astarte.realms
```

Comment:
This should be computed using ` CQLUtils.realm_name_to_keyspace_name("astarte", Config.astarte_instance_id!())` rather than just interpolating the instance id

Reply:
this is the main "*astarte" keyspace, it does not refer directly to a realm but rather to the entire instance (that can contains multiple realms), so is not correct to use
"realm_name_to_keyspace_name"